maps/submaps/surface_submaps/plains/plains_areas.dm

```diff
@@ -20,4 +20,6 @@
 	name = "Rocky2"
 
 /area/submap/Peninsula
-	name = "Peninsula"
\ No newline at end of file
+	name = "Peninsula"
+
+/area/submap/PascalB
```

Comment:
This appears to lack a name.

Reply:
Not... Really sure where that line wandered off to